src/assets/simp-adapter/build/simp-adapter.spec

```diff
@@ -214,7 +211,7 @@ puppet config set stringify_facts false || :
 (
   cd %{puppet_confdir}
 
-  simp_overrides='hiera.yaml auth.conf'
+  simp_overrides='hiera.yaml'
```

Comment:
@lnemsick-simp I will address this with https://simp-project.atlassian.net/browse/SIMP-3285, in pupmod-simp-pupmod